src/main.ts

```diff
@@ -58,6 +59,67 @@ const handleExportInventory = async () => {
     });
 };
 
+const handleImportInventory = async () => {
+  console.log('Importing inventory...');
+  
+  try {
+    // Step 1: Get import data and preview
+    const result = await importInventory(inventory, {
+      mergeStrategy: MergeStrategy.REPLACE,
+      validateSchema: true,
+      createBackup: false,
+    });
+
+    if (!result.success) {
+      if (result.error) {
+        alert(`Errore durante l'importazione: ${result.error}`);
+      }
+      return;
+    }
```

Comment:
Why nestin the if?
If it's not a success, it's an error:


```suggestion
      if (result.error) {
        alert(`Errore durante l'importazione: ${result.error}`);
        return;
      }
```